app/controllers/controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchCategories } = require("../models/model");
+
+exports.getCategories = (request, response) => {
+  fetchCategories()
+    .then((categories) => {
+      response.status(200).send({ categories: categories });
```

Comment:
Nothing wrong with this but you can refactor to the shorthand of {categories}, whenever the variable name is the same as the key you are wanting to use! :)